morango/models.py

```diff
@@ -293,6 +295,10 @@ class DatabaseMaxCounter(AbstractCounter):
     """
 
     partition = models.CharField(max_length=128, default="")
+    profile = models.CharField(max_length=40)
```

Comment:
Since we serialize based on profile, I think this field is important for tracking that. 
The empty `""` for profile could signify full serialization?
we can also just run through all the profiles and have the partition be the default value `""` to mean we serialized everything for each profile.

Reply:
Ah, this came in as I was commenting below.

> Since we serialize based on profile, I think this field is important for tracking that.

Since we don't expect partition collisions across profiles (except for empty string, for this model), we may be OK? In the empty string case, it doesn't matter which profile it came from.

Reply:
In the empty string case, it doesn't matter as long as we serialize all profiles?

Reply:
> In the empty string case, it doesn't matter as long as we serialize all profiles?

Even if we've only serialized one of multiple profiles, the existence of `(instance_id=X, partition="", counter=554)` in the DMC table means that we have _everything_ locally for that instance, up through that counter. As far as I'm able to reason it through, it shouldn't matter what profile the serializations happened under, as each one will increment that counter for the same instance_id.

Reply:
(Because there can't be a higher counter for another profile, since it all happened in this DB)

Reply:
(And if another profile hasn't been serialized in a long time, then the changes it has locally in the app layer aren't yet even associated with a counter, so they won't get skipped when we serialized them later, as they'll get new higher counters then)